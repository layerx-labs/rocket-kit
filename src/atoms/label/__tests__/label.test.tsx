import React from 'react';
import { render, screen } from '@testing-library/react';
import { Label } from '../../..';

describe('Label', () => {
  it('renders', () => {
    const { asFragment } = render(<Label value={'Amount'} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('has correct value', async () => {
    const fieldText = 'Savings';
    render(<Label value={fieldText} />);
    await screen.getByText(fieldText);
  });

  it('has correct value with KAI', async () => {
    const fieldText = 'Savings';
    render(<Label value={fieldText} kai />);
    await screen.getByText(fieldText);
  });
});
