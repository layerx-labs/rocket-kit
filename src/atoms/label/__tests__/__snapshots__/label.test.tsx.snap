// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders 1`] = `
<DocumentFragment>
  <label
    class="sc-pFZIQ lkbNLm label"
  >
    Amount
  </label>
</DocumentFragment>
`;
